sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -1,5 +1,12 @@
 namespace Azure.Storage.Queues
 {
+    public partial class InvalidMessageEventArgs : Azure.SyncAsyncEventArgs
+    {
+        public InvalidMessageEventArgs(Azure.Storage.Queues.QueueClient queue, Azure.Storage.Queues.Models.PeekedMessage message, bool runSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public InvalidMessageEventArgs(Azure.Storage.Queues.QueueClient queue, Azure.Storage.Queues.Models.QueueMessage message, bool runSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public object Message { get { throw null; } }
```

Comment:
This is unfortunate but I don't have a better alternative. Two separate properties maybe?

Reply:
see comment above.